cmd/operator/deploy/crds/monitoring.googleapis.com_operatorconfigs.yaml

```diff
@@ -105,15 +105,31 @@ spec:
             type: string
           managedAlertmanager:
             default:
-              configSecret: managed-alertmanager-config
+              configSecret:
+                key: alertmanager.yaml
+                name: alertmanager
             description: ManagedAlertmanager holds information for configuring the
               managed instance of Alertmanager.
             properties:
               configSecret:
                 description: ConfigSecret refers to the name of a single-key Secret
                   in the public namespace that holds the managed Alertmanager config
                   file.
-                type: string
+                properties:
+                  key:
+                    description: The key of the secret to select from.  Must be a
+                      valid secret key.
+                    type: string
+                  name:
+                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                      TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
should this be a comment?

Reply:
This is actually [pulled in](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/24a44d0cf189f37126bda692c4c74e272ff24b2e/vendor/k8s.io/api/core/v1/types.go#L5609) from the vendored k8s api go library.